tasks/kafka-install.yml

```diff
@@ -13,6 +13,7 @@
   shell: gpg --verify {{ kafka_sig_tmp }} {{ kafka_bin_tmp }}
   ignore_errors: yes
   register: verify
+  changed_when: False
```

Comment:
i think there is this idempotence thing most roles aspire to. if u look at most tested roles they always re-run the playbook and assert that `changed=0` (e.g. https://github.com/playlist-ansible/zookeeper/blob/master/.travis.yml#L23-L27).

in some cases like this one here we just gather facts that don't actually change system state so we mask that w/ `changed_when:`.